lib/zipkin-tracer/trace.rb

```diff
@@ -199,12 +162,29 @@ def to_i; @i128; end
 
   # A span may contain many annotations
   class Span
-    attr_accessor :name, :annotations, :binary_annotations, :debug
+    module Tag
+      METHOD = "http.method".freeze
+      PATH = "http.path".freeze
+      STATUS = "http.status".freeze
+      LOCAL_COMPONENT = "lc".freeze
```

Comment:
ls is no longer needed unless there's a value for it. sometimes we had "lc"->"" just to add an endpoint to a span.

Reply:
It seems we are still using it in our library. Added TODO in https://github.com/openzipkin/zipkin-ruby/pull/141/commits/2cd021ef2861135539effbcd8d9088c48d308a81